crates/uv/tests/pip_tree.rs

```diff
@@ -330,16 +328,9 @@ fn depth() {
     "###
     );
 
-    uv_snapshot!(context.filters(), Command::new(get_bin())
-        .arg("pip")
-        .arg("tree")
-        .arg("--cache-dir")
-        .arg(context.cache_dir.path())
+    uv_snapshot!(context.filters(), context.pip_tree()
         .arg("--depth")
-        .arg("0")
-        .env("VIRTUAL_ENV", context.venv.as_os_str())
-        .env("UV_NO_WRAP", "1")
-        .current_dir(&context.temp_dir), @r###"
+        .arg("0"), @r###"
```

Comment:
These tests broke for some reason — I don't really understand why but they're written incorrectly.

Reply:
(This was a bug that I fixed in a subsequent commit)